docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Teleport with GitHub.
+description: How to use SSH key extensions with Teleport and GitHub.
+h1: Use SSH Key extensions.
+---
+
+## Prerequisites
+
+- Teleport v(=teleport.version) Open Source or Enterprise.
```

Comment:
```suggestion
- Teleport v(=teleport.version=) Open Source or Enterprise.
```

Reply:
Should the reader also have a GitHub account? Does the account require certain permissions or a certain edition of GitHub?